src/web/calculators/flywheel/index.ts

```diff
@@ -54,6 +54,7 @@ export const FlywheelStateV2Defaults = {
 export const FlywheelParamsV2 = {
   motor: withDefault(MotorParam, FlywheelStateV2Defaults.motor),
   motorRatio: withDefault(RatioParam, FlywheelStateV2Defaults.motorRatio),
+  efficiency: withDefault(NumberParam, 100),
```

Comment:
Since we are only adding a new url parameter, it's ok to not increment the url param version numbers (we would if we were renaming/removing other url params). You will need to add a default efficiency for the conversion to v1 to v2 params in `src/web/calculators/flywheel/converter.ts` though - just copy the way that current limit was done.